### PR TITLE
SDLPoP port 

### PR DESCRIPTION
**Pull request by Kegg1701**
Prince of Persia SDL port for Emuelec

### CODE REVIEW

File: packages/sx05re/emulators/sdlpop/package.mk [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+# Copyright (C) 2020-present Shanti Gilbert (https://github.com/shantigilbert)
+
+PKG_NAME="sdlpop"
+PKG_VERSION="12942c090ca653441c820b2abc8313ab748c0d24"
+PKG_ARCH="any"
+PKG_LICENSE="CUSTOM"
```
Comment (shantigilbert) [inline packages/sx05re/emulators/sdlpop/package.mk]: According to the git for SDLpop this should be GPL3

File: packages/sx05re/emuelec/config/emuelec/ports/SDLPoP.sh [hunk 2]
```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# SPDX-License-Identifier: GPL-2.0-or-later
+# Copyright (C) 2020-present Shanti Gilbert (https://github.com/shantigilbert)
+
+# Source predefined functions and variables
+. /etc/profile
+
+JSLISTENCONF="/emuelec/configs/jslisten.cfg"
```
Comment (shantigilbert) [inline packages/sx05re/emuelec/config/emuelec/ports/SDLPoP.sh]: Can you check how ports are done now, 

https://github.com/EmuELEC/EmuELEC/blob/6d4e92b4852d1bfc013ee1f09ae9ea197b32c989/packages/sx05re/emuelec/config/emuelec/ports/Diablo.sh

and change accordingly

Comment (Kegg1701) [reply packages/sx05re/emuelec/config/emuelec/ports/SDLPoP.sh]: and done

File: packages/sx05re/emulators/sdlpop/package.mk (hunk 1 above)
Comment (Kegg1701) [reply packages/sx05re/emulators/sdlpop/package.mk]: ok done

**Comment by shantigilbert** [comment]
Thank you!

**Pull request merged**